python/main.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+import xpring
+
+
```

Comment:
super nit: extra blank line

Reply:
This line is part of the Google style, but I'll remove it.